guacamole/src/main/webapp/translations/zh.json

```diff
@@ -490,31 +648,40 @@
 
     },
 
+
+
     "PROTOCOL_TELNET" : {
 
-        "FIELD_HEADER_BACKSPACE"      : "退格键发送：",
-        "FIELD_HEADER_COLOR_SCHEME"   : "配色方案：",
-        "FIELD_HEADER_CREATE_RECORDING_PATH" : "自动建立录像目录：",
-        "FIELD_HEADER_CREATE_TYPESCRIPT_PATH" : "自动建立打字稿目录：",
-        "FIELD_HEADER_FONT_NAME"      : "字体名：",
-        "FIELD_HEADER_FONT_SIZE"      : "字体大小：",
-        "FIELD_HEADER_HOSTNAME"       : "主机名：",
-        "FIELD_HEADER_USERNAME"       : "用户名：",
-        "FIELD_HEADER_PASSWORD"       : "密码：",
-        "FIELD_HEADER_PASSWORD_REGEX" : "密码规则正则表达式：",
-        "FIELD_HEADER_PORT"           : "端口：",
-        "FIELD_HEADER_READ_ONLY"      : "只读：",
-        "FIELD_HEADER_RECORDING_EXCLUDE_MOUSE"  : "排除鼠标：",
-        "FIELD_HEADER_RECORDING_EXCLUDE_OUTPUT" : "排除图像/数据流：",
-        "FIELD_HEADER_RECORDING_INCLUDE_KEYS"   : "包含按键事件：",
-        "FIELD_HEADER_RECORDING_NAME" : "录像名：",
-        "FIELD_HEADER_RECORDING_PATH" : "录像路径：",
-        "FIELD_HEADER_TYPESCRIPT_NAME" : "打字稿名：",
-        "FIELD_HEADER_TYPESCRIPT_PATH" : "打字稿路径：",
+        "FIELD_HEADER_BACKSPACE"      : "发送退格键:",
+        "FIELD_HEADER_COLOR_SCHEME"   : "配色方案:",
+        "FIELD_HEADER_CREATE_RECORDING_PATH" : "自动创建记录路径:",
+        "FIELD_HEADER_CREATE_TYPESCRIPT_PATH" : "自动创建typescript路径:",
+        "FIELD_HEADER_DISABLE_COPY"   : "禁止从终端复制：",
+        "FIELD_HEADER_DISABLE_PASTE"  : "禁用从客户端粘贴：",
+        "FIELD_HEADER_FONT_NAME"      : "字体名称:",
+        "FIELD_HEADER_FONT_SIZE"      : "字体大小:",
+        "FIELD_HEADER_HOSTNAME"       : "主机名:",
+        "FIELD_HEADER_LOGIN_FAILURE_REGEX" : "登录失败正则表达式:",
+        "FIELD_HEADER_LOGIN_SUCCESS_REGEX" : "登录成功正则表达式:",
+        "FIELD_HEADER_USERNAME"       : "用户名:",
+        "FIELD_HEADER_USERNAME_REGEX" : "用户名正则表达式:",
+        "FIELD_HEADER_PASSWORD"       : "密码:",
+        "FIELD_HEADER_PASSWORD_REGEX" : "密码正则表达式:",
+        "FIELD_HEADER_PORT"           : "端口:",
+        "FIELD_HEADER_READ_ONLY"      : "只读:",
+        "FIELD_HEADER_RECORDING_EXCLUDE_MOUSE"  : "排除鼠标:",
+        "FIELD_HEADER_RECORDING_EXCLUDE_OUTPUT" : "排除图形/流:",
+        "FIELD_HEADER_RECORDING_INCLUDE_KEYS"   : "包含关键事件:",
+        "FIELD_HEADER_RECORDING_NAME" : "记录名称:",
+        "FIELD_HEADER_RECORDING_PATH" : "记录路径:",
+        "FIELD_HEADER_SCROLLBACK"     : "最大回滚尺寸:",
+        "FIELD_HEADER_TERMINAL_TYPE"   : "终端类型:",
+        "FIELD_HEADER_TYPESCRIPT_NAME" : "Typescript 名称:",
+        "FIELD_HEADER_TYPESCRIPT_PATH" : "Typescript 路径:",
 
         "FIELD_OPTION_BACKSPACE_EMPTY" : "",
-        "FIELD_OPTION_BACKSPACE_8"     : "退格键（Ctrl-H）",
-        "FIELD_OPTION_BACKSPACE_127"   : "删除键（Ctrl-?）",
+        "FIELD_OPTION_BACKSPACE_8"     : "Backspace (Ctrl-H)",
+        "FIELD_OPTION_BACKSPACE_127"   : "Delete (Ctrl-?)",
```

Comment:
Why are these being changed back into English?

Reply:
Okay,I correct it.